modules/3d-tiles/src/index.ts

```diff
@@ -2,6 +2,7 @@
 export {Tiles3DLoader} from './tiles-3d-loader';
 export {CesiumIonLoader} from './cesium-ion-loader';
 export {Tile3DSubtreeLoader} from './tile-3d-subtree-loader';
+export {TZ3Loader, TZ3LoaderOptions} from './tz3-loader';
```

Comment:
Nit: Split type and symbol exports. 
```suggestion
export type {TZ3LoaderOptions} from './tz3-loader';
export {TZ3Loader} from './tz3-loader';
```
Thanks for adding loader types here. It would be nice to go back to other 3d tiles loaders and add options types and export them here.